charts/cronjob/templates/external-config-secrets.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.externalConfig.enabled -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "cronjob.fullname" . }}-external-config-secret
+  labels:
+    app: {{ template "cronjob.name" . }}
+    chart: {{ template "cronjob.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+type: Opaque
+data:
+  {{ .Values.externalConfig.configFileName }}: |
```

Comment:
this multiline string makes the deployment of the datafeed exporter to fail because it can't decode the \n. Without multiline string works.

There are other people with the same issue: https://github.com/ceph/ceph-helm/pull/40/commits/6cba960c2f4fad3c3bc9153dec1989f0d1b1cd75

Reply:
https://github.com/commercetools/k8s-charts/pull/23/commits/0664497e64454c716a82a55bef8c788244c12460